model_zoo/census_model_sqlflow/wide_and_deep/transform_ops.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+
+
+class TransformOpType(Enum):
+    HASH = (1,)
+    BUCKETIZE = (2,)
+    LOOKUP = (3,)
+    EMBEDDING = (4,)
+    CONCAT = (5,)
+    ARRAY = 6
+
+
+class TransformOp(object):
+    def __init__(self, name, input, output):
+        self.name = name
+        # The input name or a list of input names
+        self.input = input
+        # The output name
+        self.output = output
+        # The type of the TransformOp.
+        # The value is one of `TransformOpType`.
+        self.op_type = None
+
+
+class Vocabularize(TransformOp):
```

Comment:
Add a docstring to explain which Keras layer or feature column the transformation operator use

Reply:
Good idea.